ynput/OpenPype
# Applications: Attributes creation

## Changelog Description
Applications addon does not cause infinite server restart loop.

## Additional info
Applications addon creates set of all applications and tools from all versions and all settings. Also changes behavior of old version to they can live next each other -> that is achieved by `pre_setup` method which is ATM available only in latest dev server.

This will keep all applications from all versions of applications addon in attributes (not easy to remove).

## Testing notes:
1. Create packages
2. Upload `applications` addon to server
3. Restart server
4. It should not cause infinite restart of server
5. Choose the addon in a bundle and set the bundle to production
6. Go to settings and add an application variant (anywhere)
7. Restart server
8. It should not cause infinite restart of server and the application should be in attributes
